### ISSUE TITLE
Drag'n drop svg to interface not working?

### ISSUE THREAD

**Issue by Chelvis** [fontello/fontello#735]
Hello!

I'm trying to drop svg files in fontello's interface to include it as an icon in my library, but i'ts not working today. The cursor becomes as blocked and I can add any icon from my svg files.

Anyone with a solution?

**Comment by puzrin** [comment]
Try different browser. Also, you can use dropdown menu in toolbar -> `import`

**Comment by Chelvis** [comment]
> Try different browser. Also, you can use dropdown menu in toolbar -> `import`

Yep, it appears to be an issue only in chrome. In Edge and Firefox stills working fine. I used the import tool, but i like de drag'n drop resource. Anyway, thank you so much!

**Comment by rlidwka** [comment]
> In Edge and Firefox keeps working fine.

Edge and Chrome are different UIs slapped onto the same browser engine (Chromium). They should work identically.

Anyways, drag and drop works for me in both Chrome 91 and Firefox 90. This may be related to chrome version, which one are you running? Please try again in the latest version, maybe its fixed itself.
